Description:
 Increase PJ_ICE_MAX_CAND to 256. Upstream's default, 16, is
 not enough to handle cases with numerous local/remote candidates
 like the case when multiple hosts in IPv6 and IPv4 exists.
 Tests show that 40 entries are needed with 5 local host IPs.
 .
 Increase PJ_ICE_MAX_CHECKS, which would be way too low for the
 number of candidates.
 .
 Increase PJ_ICE_MAX_STUN to 3:
  - One for IPv4
  - One for IPv6
  - One set by the user
 .
 Increase PJ_ICE_COMP_BITS so that we can have more components:
  - 2 Audio components
  - 2 Video components
Author: Guillaume Roguez <guillaume.roguez@savoirfairelinux.com>

--- a/pjnath/include/pjnath/config.h
+++ b/pjnath/include/pjnath/config.h
@@ -233,3 +233,3 @@
 #ifndef PJ_ICE_MAX_CAND
-#   define PJ_ICE_MAX_CAND			    16
+#   define PJ_ICE_MAX_CAND			    256
 #endif
@@ -243,3 +243,3 @@
 #ifndef PJ_ICE_ST_MAX_CAND
-#   define PJ_ICE_ST_MAX_CAND			    8
+#   define PJ_ICE_ST_MAX_CAND			    32
 #endif
@@ -254,3 +254,3 @@
 #ifndef PJ_ICE_MAX_STUN
-#   define PJ_ICE_MAX_STUN			    2
+#   define PJ_ICE_MAX_STUN			    3
 #endif
@@ -274,3 +274,3 @@
 #ifndef PJ_ICE_COMP_BITS
-#   define PJ_ICE_COMP_BITS			    1
+#   define PJ_ICE_COMP_BITS			    2
 #endif
@@ -325,3 +325,3 @@
 #ifndef PJ_ICE_MAX_CHECKS
-#   define PJ_ICE_MAX_CHECKS			    32
+#   define PJ_ICE_MAX_CHECKS			    150
 #endif
